contrib/endpoints/src/api_manager/BUILD

```diff
@@ -164,7 +174,7 @@ cc_test(
     ],
     linkstatic = 1,
     deps = [
-        ":api_manager",
+        ":http_template",
```

Comment:
`HttpTemplate` doesn't have any dependency so it is easier to be separated, `PathMatcher` needs a lot of refactoring.